editor/test/integration/gui_test.clj

```diff
@@ -1149,23 +1149,23 @@
+              (is (has-successor? [referenced-scene-text :trivial-gui-scene-info] [referenced-scene-text :prop->value]))
+              (is (has-successor? [referenced-scene-added-text :trivial-gui-scene-info] [referenced-scene-added-text :prop->value])))
 
             (testing "Successors in referencing scene."
-              (is (has-successor? [referencing-scene :visible-layout] [referencing-scene :current-layout]))
-              (is (has-successor? [referencing-scene :current-layout] [referencing-scene-node-tree :current-layout]))
-              (is (has-successor? [referencing-scene-node-tree :current-layout] [referencing-scene-button :current-layout]))
-              (is (has-successor? [referencing-scene-button :current-layout] [referencing-scene-referenced-scene :current-layout]))
-              (is (has-successor? [referencing-scene-referenced-scene :current-layout] [referencing-scene-referenced-scene-node-tree :current-layout]))
-              (is (has-successor? [referencing-scene-referenced-scene-node-tree :current-layout] [referencing-scene-referenced-scene-text :current-layout]))
-              (is (has-successor? [referencing-scene-referenced-scene-node-tree :current-layout] [referencing-scene-referenced-scene-added-text :current-layout]))
-              (is (has-successor? [referencing-scene-referenced-scene-text :current-layout] [referencing-scene-referenced-scene-text :prop->value]))
-              (is (has-successor? [referencing-scene-referenced-scene-added-text :current-layout] [referencing-scene-referenced-scene-added-text :prop->value])))
+              (is (has-successor? [referencing-scene :visible-layout] [referencing-scene :trivial-gui-scene-info]))
+              (is (has-successor? [referencing-scene :trivial-gui-scene-info] [referencing-scene-node-tree :trivial-gui-scene-info]))
+              (is (has-successor? [referencing-scene-node-tree :trivial-gui-scene-info] [referencing-scene-button :trivial-gui-scene-info]))
+              (is (has-successor? [referencing-scene-button :template-trivial-gui-scene-info] [referencing-scene-referenced-scene :trivial-gui-scene-info]))
+              (is (has-successor? [referencing-scene-referenced-scene :trivial-gui-scene-info] [referencing-scene-referenced-scene-node-tree :trivial-gui-scene-info]))
+              (is (has-successor? [referencing-scene-referenced-scene-node-tree :trivial-gui-scene-info] [referencing-scene-referenced-scene-text :trivial-gui-scene-info]))
+              (is (has-successor? [referencing-scene-referenced-scene-node-tree :trivial-gui-scene-info] [referencing-scene-referenced-scene-added-text :trivial-gui-scene-info]))
+              (is (has-successor? [referencing-scene-referenced-scene-text :trivial-gui-scene-info] [referencing-scene-referenced-scene-text :prop->value]))
+              (is (has-successor? [referencing-scene-referenced-scene-added-text :trivial-gui-scene-info] [referencing-scene-referenced-scene-added-text :prop->value])))
```

Comment:
These tests are a bit intrusive, but on the other hand they can be quite helpful to diagnose issues.